desktop/cmp/panel/Panel.js

```diff
@@ -106,7 +108,7 @@ export const [Panel, panel] = hoistCmp.withFactory({
                 style: {display: collapsed ? 'none' : 'flex'},
                 items: [
                     parseToolbar(tbar),
-                    ...Children.toArray(children),
+                    parseError(error, contextModel) ?? Children.toArray(children),
```

Comment:
I expect we still need to spread the Children array in the normal (non-error) case - otherwise wouldn't React want to insist upon a key?

Reply:
I made this change with Dan as part of a refactor, but I have to admit it was not intentional.

It works fine as is, relying on internal React behavior to flatten child arrays.    Not sure what is better -- we certainly don't rely on that flattening behavior in our code right now but we could .